winui/Kanban/swimlane.md

```diff
@@ -0,0 +1,220 @@
+        taskDetail.Id = "4";
+        taskDetail.Assignee = "Janet Leverling";
+        taskDetail.Description = "Fix the issues reported in the IE browser.";
+        taskDetail.Category = "Review";
+        taskDetail.IndicatorColorKey = "Normal";
+        taskDetail.Image = new Image
+        {
+            Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle1.png"))
+        };
+
+        taskDetails.Add(taskDetail);
+
+        taskDetail = new KanbanModel();
+        taskDetail.Title = "Application performance";
+        taskDetail.Id = "2";
+        taskDetail.Assignee = "Andrew Fuller";
+        taskDetail.Description = "Improve application performance.";
+        taskDetail.Category = "Backlog";
+        taskDetail.IndicatorColorKey = "Normal";
+        taskDetail.Image = new Image
+        {
+            Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle2.png"))
+        };
+
+        taskDetails.Add(taskDetail);
+
+        taskDetail = new KanbanModel();
+        taskDetail.Title = "Analysis";
+        taskDetail.Id = "18";
+        taskDetail.Assignee = "Andrew Fuller";
+        taskDetail.Description = "Analyze SQL server 2008 connection.";
+        taskDetail.Category = "In Progress";
+        taskDetail.IndicatorColorKey = "Low";
+        taskDetail.Image = new Image
+        {
+            Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle2.png"))
+        };
+
+        taskDetails.Add(taskDetail);
+
+        taskDetail = new KanbanModel();
+        taskDetail.Title = "Responsive support";
+        taskDetail.Id = "14";
+        taskDetail.Assignee = "Laura Callahan";
+        taskDetail.Description = "Add responsive support to application.";
+        taskDetail.Category = "In Progress";
+        taskDetail.IndicatorColorKey = "High";
+        taskDetail.Image = new Image
+        {
+            Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle4.png"))
+        };
+
+        taskDetails.Add(taskDetail);
+
+        taskDetail = new KanbanModel();
+        taskDetail.Title = "Filtering feature";
+        taskDetail.Id = "48";
+        taskDetail.Assignee = "Laura Callahan";
+        taskDetail.Description = "Check filtering validation.";
+        taskDetail.Category = "Testing";
+        taskDetail.IndicatorColorKey = "Normal";
+        taskDetail.Image = new Image
+        {
+            Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle4.png"))
+        };
+
+        taskDetails.Add(taskDetail);
+        return taskDetails;
+    }
+
+    #endregion
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customize swim lane header
+
+The WinUI Kanban control allows you to customize the appearance of swimlane headers using the SwimlaneHeaderTemplate property. This enables you to create a unique design for the header displayed at the start of each swimlane group.
+
+The following code example demonstrates how to customize a swimlane header:
```

Comment:
The following code example demonstrates how to customize a swimlane header.

Reply:
Changed.